clang-tools-extra/clang-doc/HTMLGenerator.cpp

```diff
@@ -653,15 +697,35 @@ static std::vector<std::unique_ptr<TagNode>>
 genHTML(const EnumInfo &I, const ClangDocContext &CDCtx) {
   std::vector<std::unique_ptr<TagNode>> Out;
   std::string EnumType = I.Scoped ? "enum class " : "enum ";
+  // Determine if enum members have comments attached
+  bool HasComments = false;
+  for (const auto &M : I.Members) {
+    if (!M.Description.empty()) {
+      HasComments = true;
+      break;
+    }
+  }
```

Comment:
```suggestion
  bool HasComments = std::any_of(I.Members.begin(), I.Members.end(), [](auto M){return M.Description.empty();});
```
Maybe something like this instead? I didn't check the syntax, so take w/ a grain of salt.